### PR TITLE
JM GetRoute Weather [JM-1]

### PR DESCRIPTION
**Pull request by jm220**
* **Please check if the PR fulfills these requirements**
- [x] The pull request title contains the ticket number in the title. For example [CB-1]

* **What kind of change does this PR introduce?** (Bug fix, new feature...)
edited main index.js by adding GET route

* **What is the current behavior?What does the code do before you made the new changes?***
Current behavior has no GET/POST routes to main branch

* **What is the new behavior?**
Displays the current weather

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)
N/A

* **Other information**:
Displays weather in Degrees F.

I accidentally deleted JM/index.js, therefore it appears as if I had modified that file when I added it again.
However, the file is the same as in the other PR.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+const app = express()
+const axios = require('axios')
+app.use(express.json())
+
+app.get('/getWeather', function(req, res){
+    const url = 'https://codubee-projects-api.herokuapp.com/getWeather'
+    axios.get(url)
+        .then((response) => {
+            console.log(response.data)
+            res.status(200).json({"Temperature is: ": response.data + " F"})
```
Comment (Codubee) [inline index.js]: Can the message here just be "temp":"76". The key "Temperature is:" is a bit of an odd key to have.

**Review by Codubee — CHANGES_REQUESTED** [review]
LGTM! Just one change.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        })
+        .catch((error) => {
+            console.log(error);
+            postmanresponse.json({"Error": "Couldn't load weather"})
```
Comment (arsoljon) [inline index.js]: on line 15, I am assuming 'postmanresponse' should be 'res' because 'postmanresponse' was never initialized.

**Review by arsoljon — COMMENTED** [review]
One changed need on line 15.

File: index.js [hunk 3]
```diff
@@ -8,11 +8,11 @@ app.get('/getWeather', function(req, res){
     axios.get(url)
         .then((response) => {
             console.log(response.data)
-            res.status(200).json({"Temperature is: ": response.data + " F"})
+            res.status(200).json({"Temp: ": response.data})
```
Comment (arsoljon) [inline index.js]: You don't need the colon included in the key. Only the one in the middle, separating the key and value.

**Review by arsoljon — COMMENTED** [review]
Line 11, adjust the key.

**Review by arsoljon — DISMISSED** [review]
Looks good.

**Review by Codubee — DISMISSED** [review]

**Pull request merged**